hooks/python-pkgs.txt

```diff
@@ -0,0 +1,5 @@
+-e git+https://github.com/whitmo/charmhelpers.git#egg=charmhelpers
```

Comment:
These are scary to me, I dont look at these files often. Do we have open PR's to get these into upstream?